### PR TITLE
INJI-131 | Translate labels 

### PR DESCRIPTION
**Pull request by KiruthikaJeyashankar**
The missed labels (Card, Verifier, Wallet, Bluetooth, Selfie) have been translated

### CODE REVIEW

File: locales/hin.json [hunk 1]
```diff
@@ -384,7 +387,7 @@
         },
         "accepted": {
           "title": "सफलता!",
-          "message": "आपका {{vcLabel}} को {{रिसीवर}} के साथ सफलतापूर्वक साझा कर दिया गया है"
```
Comment (vijay151096) [inline locales/hin.json:387]: Translation for {{vclabel}} and {{receiver}} is missing

File: screens/Home/MyVcs/DownloadingVcModalController.ts [hunk 2]
```diff
@@ -1,13 +1,3 @@
-import { useSelector } from '@xstate/react';
-import { useContext } from 'react';
-import { selectVcLabel } from '../../../machines/settings';
-import { GlobalContext } from '../../../shared/GlobalContext';
-
 export function useDownloadingVcModal() {
```
Comment (vijay151096) [inline screens/Home/MyVcs/DownloadingVcModalController.ts:1]: If this useDownloadingVCModel() is not used anywhere can we remove this ?

File: locales/hin.json (hunk 1 above)
Comment (KiruthikaJeyashankar) [reply locales/hin.json:387]: This translation SendVcScreen:status is not available in en.json locale file. Only the translations that were available in en.json were translated.

File: locales/ara.json [hunk 3]
```diff
@@ -64,12 +66,12 @@
     "ok": "تمام",
     "errors": {
       "savingFailed": {
-        "title": "فشل حفظ {{vcLabelSingular}}",
```
Comment (vijay151096) [inline locales/ara.json:67]: Are we not replacing the {{placeholder}} with the {{native_translations}} ? For Arabic alone, the order itself is changed. any specific reason for arabic ?

File: screens/Home/MyVcs/DownloadingVcModalController.ts (hunk 2 above)
Comment (KiruthikaJeyashankar) [reply screens/Home/MyVcs/DownloadingVcModalController.ts]: Removed since it is not used anywhere

File: locales/ara.json [hunk 4]
```diff
@@ -149,8 +151,8 @@
     "qstnMarkToolTip": "معرف التطبيق متاح في الإقرار الذي تم استلامه بعد التسجيل."
   },
   "IdInputModal": {
-    "header": "أدخل UIN / VID لتنزيل {{vcLabel}}",
```
Comment (vijay151096) [inline locales/ara.json:152]: As Mentioned in the PR description UIN / VID should also be translated, but the translations are missing here. 
1. Either UIN / VID has to be translated post confirming with Pragya. 
2. or update the PR Description.

File: locales/hin.json (hunk 1 above)
Comment (vijay151096) [reply locales/hin.json:387]: Check with bhargavi, all the missed translation have been added. if it is used and missed translations,
1. please add the needed translations .
2. if it not used, please remove the unused literals.

File: locales/ara.json (hunk 4 above)
Comment (KiruthikaJeyashankar) [reply locales/ara.json:152]: > As Mentioned in the PR description UIN / VID should also be translated, but the translations are missing here.
> 
> 1. Either UIN / VID has to be translated post confirming with Pragya.
> 2. or update the PR Description.

Apologies for the wrong update in PR description, as a part of this card only the constants like Card, Verifier, Wallet, Bluetooth, Selfie will be translated. Have updated PR description with the same

File: locales/hin.json (hunk 1 above)
Comment (KiruthikaJeyashankar) [reply locales/hin.json:387]: if it is used and missed translations -> please add the needed translations => This has been done in this PR.
For removing unused literals, will connect with @PuBHARGAVI once and will do the needful

Comment (KiruthikaJeyashankar) [reply locales/hin.json:387]: Update: all the {{vcLabel}} has been translated in all locales file except English

**Pull request merged**